src/hutch_bunny/cli.py

```diff
@@ -36,12 +36,20 @@ def main() -> None:
     # Bunny passed args.
     args = parser.parse_args()
 
+
     with open(args.body) as body:
         query_dict = json.load(body)
-    results_modifier = get_results_modifiers_from_str(args.results_modifiers)
+
+    modifiers_list = results_modifiers(
```

Comment:
I think this should be reverted, as the CLI does require the obfuscation modifiers to be passed through `args` instead.

The background is, there are 2 ways to run Bunny: 
1. Through the daemon (as a service to RQUEST)
2. Through the CLI (manually).

The manual CLI is still used by Hutch-TREFX, as the container is executed by WfeXs etc. - it passes the obfuscation settings through command line, rather than environment variable. So this change would break obfuscation there.